etc/vzlogger.conf

```diff
@@ -153,6 +155,27 @@
                 "middleware": "http://localhost/middleware.php"
             }
         },
+        {
+            /* A minimal example that "just works" on Debian. 
+               The steps needed are:
+               1. Install influxdb: sudo apt install influxdb influxdb-client
+               2. Start the client by calling influx on the command line and
+               create db: CREATE DATABASE vzlogger
+             */
+            "enabled": false,               // disabled meters will be ignored
+            "allowskip": false,                  // errors when opening meter may be ignored if enabled
+
+            "protocol": "random",
+            "interval": 2,
+            "max": 40.0,                    // has to be double!
+            "min": -5.0,                    // has to be double!
+            "channel": {
+                "uuid": "25a53b28-75ec-4764-a8a6-68587722ddab",
+                "api": "influxdb",
```

Comment:
(also, currently all examples in vzlogger.conf use the volkszaehler api,
and other apis have their own example.conf , maybe it should be kept that way.)

Reply:
I can keep it in the debian branch, but I definitely want it to be in the installed vzlogger.conf.

Reply:
Unnecessary identifier removed.

Reply:
I have changed the target branch to debian, this looks like a good I idea since I found your objection reasonable. OK?

Reply:
my suggestion was rather to move this to a separate file.
but it's not critical either way.
you shouldn't have to maintain a separate branch for debian, at least not for this.